kafka-client/src/main/java/dev/responsive/kafka/internal/stores/PartitionedOperations.java

```diff
@@ -151,10 +156,15 @@ public static PartitionedOperations create(
       storeRegistry.registerStore(registration);
 
       final boolean migrationMode = ConfigUtils.responsiveMode(config) == ResponsiveMode.MIGRATE;
-      long startingTimestamp = -1;
-      final Optional<Duration> ttl = params.timeToLive();
-      if (migrationMode && ttl.isPresent()) {
-        startingTimestamp = Instant.now().minus(ttl.get()).toEpochMilli();
+      long startTimeMs = -1;
+      if (migrationMode && params.ttlProvider().isPresent()) {
+        // TODO(sophie): figure out how to account for row-level ttl in migration mode
```

Comment:
let's remove this TODO and just create a ticket since the code isn't wrong (it's just an unsupported feature at the moment)

Reply:
https://www.notion.so/responsivedev/Support-row-level-ttl-in-migration-mode-12927549be4c8065b9e1c92ed0f9030f?pvs=4